addons/beehave/nodes/beehave_root.gd

```diff
@@ -56,12 +70,17 @@ func get_last_condition_status():
 			return "RUNNING"
 	return ""
 
-
 func enable():
 	self.enabled = true
-	set_physics_process(true)
-
+	set_process(process_mode == ProcessMode.IDLE)
```

Comment:
One last thing: could we do this instead +
here?

set_proccess_mode(self.process_mode)

Reply:
Good idea!